docs/source/delta-storage.md

```diff
@@ -149,8 +149,17 @@ This section explains how to quickly start reading and writing Delta tables on S
     --conf spark.io.delta.storage.S3DynamoDBLogStore.ddb.region=us-west-2
    ```
 
+   **For ScyllaDB:**
+   ```bash
+     bin/spark-shell \
+      --packages io.delta:delta-spark_2.12:3.0.0,org.apache.hadoop:hadoop-aws:3.3.4,io.delta:delta-storage-s3-dynamodb:3.0.0 \
```

Comment:
3.0.0? not 3.1.0? this is for latest docs

Reply:
I opened this PR 2 months ago when the version was 3.0.0, anyway, I updated it to 3.1.0.